### ISSUE TITLE
[SLOPtools] no tests (Trac #1191)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1185]
There are no tests at all.  Please add some tests under `resources/test`.

Migrated from https://code.icecube.wisc.edu/ticket/1191
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "There are no tests at all.  Please add some tests under `resources/test`.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[SLOPtools] no tests", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-08-19T16:51:41", 
    "milestone": "", 
    "owner": "jacobi", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T08:15:17: 
* jacobi changed resolution from "" to "fixed"
* jacobi commented:

> In changeset 136765:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136765"
> Add test script. Closes #1185
> ```


* jacobi changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
